File: Python PowerPoint Examples/ShapesApiDemo/getSlideShapes.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SlideShapes.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
result = api.get_slide_shapes(fileName, slide_index, password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/getSlideShapes.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "SlideShapes.pptx";
$password = null;
$folder = "input";
$storage = null;

$slideIndex = 0;
$result = $apiInstance->getSlideShapes($name,$slideIndex, $password, $folder, $storage);
print_r($result);